### PR TITLE
[Merged by Bors] - Simple example for python smartmodule use

### PR DESCRIPTION
**Pull request by simlay**

### CODE REVIEW

File: embeds/client-examples/python/hello-python-smartmodule.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    producer = fluvio.topic_producer(TOPIC_NAME)
+    producer.send_string("Hello World! - Time is: {}".format(datetime.now()))
+
+    # Consume from topic
+    # We're just going to get the last record
+    consumer = fluvio.partition_consumer(TOPIC_NAME, PARTITION)
+
+
+    # Create a ConsumerConfig using your "uppercase-map" smartmodule
+    config = ConsumerCoonfig()
```
Comment (tjtelan) [inline embeds/client-examples/python/hello-python-smartmodule.py:33]: I think this is a typo?

`ConsumerCoonfig()` -> `ConsumerConfig()`

**Review by tjtelan — APPROVED** [review]
Other than typos, looks good to me!

Doesn't need to be part of this update, but we should have some standard smartmodules (hub and non-hub) that we can use in our examples so we can run this in CI

File: embeds/client-examples/python/hello-python-smartmodule.py (hunk 1 above)
Comment (simlay) [reply embeds/client-examples/python/hello-python-smartmodule.py:33]: Ah. Correct.

**Pull request closed without merging**